runescape-api/src/main/java/net/runelite/rs/api/RSMenuAction.java

```diff
@@ -24,6 +24,22 @@
  */
 package net.runelite.rs.api;
 
+import net.runelite.mapping.Import;
+
 public interface RSMenuAction
 {
+	@Import("action")
+	void setOption(String yes);
+
+	@Import("opcode")
+	void setOpcode(int yes);
+
+	@Import("argument0")
+	void setIdentifier(int yes);
+
+	@Import("argument1")
+	void setArg1(int yes);
+
+	@Import("argument2")
+	void setArg2(int yes);
```

Comment:
Current naming scheme is param0, and param1. Is there a purpose for these being different?

Reply:
Cause they already were like this and changing is effort, fee; free to do tho